spec/rails_helper.rb

```diff
@@ -21,15 +21,6 @@
 # require only the support files necessary.
 #
 # Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
-
-# Checks for pending migrations and applies them before tests are run.
```

Comment:
This project use ridgepole so do not use migration.

by the way, do there need ridgepole schema?